.circleci/config.yml

```diff
@@ -297,9 +297,15 @@ jobs:
           name: Build Homepage
           command: |
             python3 tools/scripts/synchronize_docs.py
-            (cd examples/stm32f469_discovery/blink; lbuild -D ::generate_module_docs=True build -m "modm:**")
-            (cd docs/src/reference && ln -s ../../../examples/stm32f469_discovery/blink/modm/docs/module)
+            python3 tools/scripts/generate_module_docs.py
             (cd docs && mkdocs build)
+      # FIXME: Allow localhost:port to work with Docker
+      # - run:
+      #     name: Check Homepage
+      #     command: |
+      #       apt install -y nodejs npm
+      #       npm install -g broken-link-checker-local
+      #       blcl modm/docs/modm.io/docs -ro --host-requests 100 --filter-level 1
```

Comment:
This would be super useful, but `blcl` opens up a random port on `localhost` and the Docker container doesn't allow that…

Reply:
I might be asking stupid  things, but can't `blcl` open a unix socket and/or use something like [appswitch](http://appswitch.io/) to make networking easier?
appswitch, if understand it correctly, rewrites every `bind` and `connect` to tcp. Locally, it even replaces it with unix sockets, making it easier to configure and perform quicker (due to less networking overhead).

Reply:
My issue was that I don't know enough about Docker/Containers/CircleCI to fix this and a quick googling of the issue didn't yield an *simple* solutions.

The check is also only *somewhat* useful, since a few links are only valid with sign-in (the CircleCI links) and some links to servers are overwhelmed ([Blue Pill](http://wiki.stm32duino.com/forums.html?title=Blue_Pill)). So this could only be an optional step anyways (I mean, gosh the internet is fickle…)